db/migrations/009.do.add-payload-retrievability-column.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE active_deals
+ADD COLUMN payload_unretrievable BOOLEAN,
+ADD COLUMN last_payload_retrieval TIMESTAMP WITH TIME ZONE;
```

Comment:
Have you tried running this migration with type set to `TIMESTAMPZ`?

```suggestion
ADD COLUMN last_payload_retrieval TIMESTAMPZ;
```

Reply:
I have yes, I think it is more intuitive with the full description. `TIMESTAMPZ` is not a word that is usually used and thus not have any real meaning. Using the full description makes it self explainatory.